### ISSUE TITLE
SVG Support?

### ISSUE THREAD

**Issue by SSIvanov19** [TheCherno/Walnut#6]
Is there any way to load .svg image?

**Comment by codename-irvin** [comment]
@SSIvanov19 I don't believe that ImGui supports rendering SVGs and it doesn't look like Walnut has added anything to support this. For icons, ImGui recommends using icon fonts (see https://fonts.google.com/icons for an example).

However, I agree that support for this would be very useful.

**Comment by ZanzyTHEbar** [comment]
I second this - i would love to have SVG support. Right now images look pixalated when resized to fit the veiwport.
